docs/basics/rules.md

```diff
@@ -550,7 +550,7 @@ tags:
 
   # CVE
   - cve.2022.27925 # OR
```

Comment:
This format is deprecated. Can you please remove the example with the dots. I'm AFK for now.

Reply:
Sure thing, done.